src/Code.js

```diff
@@ -469,7 +469,12 @@ function MBCUSTOMQUERY(events, groupBy, orderBy, limit, offset) {
 
   // turn the array of objects into a flat array
   const objArr = results.result.rows;
-  return objectArrayToArray(objArr);
+  const headersPreset = Array(payload.events[0].select.length);
+  // eslint-disable-next-line no-restricted-syntax, guard-for-in
+  for (const s of payload.events[0].select) {
```

Comment:
Note that it's not a requirement that each event row is the same event, nor that it has the same aliases. Therefore, I don't think we can just use `events[0]`:

For example, this is valid:
![image](https://user-images.githubusercontent.com/5861379/103096221-936d4900-4646-11eb-9658-5e56ac01d269.png)

Which generates the following query:

```json
{
    "events": [
        {
            "eventName": "Transfer",
            "select": [
                {
                    "alias": "abc",
                    "inputIndex": 0,
                    "type": "input"
                },
                {
                    "alias": "ghi",
                    "inputIndex": 2,
                    "type": "input"
                }
            ],
            "filter": {
                "rule": "and",
                "children": [
                    {
                        "operator": "equal",
                        "value": "autotoken",
                        "fieldType": "contract_address_label"
                    }
                ]
            }
        },
        {
            "eventName": "Mint",
            "select": [
                {
                    "alias": "def",
                    "inputIndex": 1,
                    "type": "input"
                },
                {
                    "alias": "jkl",
                    "inputIndex": 2,
                    "type": "input"
                }
            ],
            "filter": {
                "rule": "and",
                "children": [
                    {
                        "operator": "equal",
                        "value": "autotoken",
                        "fieldType": "contract_address_label"
                    }
                ]
            }
        }
    ]
}
```

With the following result:

```
{
    "status": 200,
    "message": "success",
    "result": {
        "rows": [
            {
                "abc": null,
                "def": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "ghi": null,
                "jkl": "100000000000"
            },
            {
                "abc": "0x0000000000000000000000000000000000000000",
                "def": null,
                "ghi": "100000000000",
                "jkl": null
            },
            {
                "abc": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "def": null,
                "ghi": "10000000000",
                "jkl": null
            },
            {
                "abc": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "def": null,
                "ghi": "1000000000",
                "jkl": null
            },
            {
                "abc": null,
                "def": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "ghi": null,
                "jkl": "100000000000"
            },
            {
                "abc": "0x0000000000000000000000000000000000000000",
                "def": null,
                "ghi": "100000000000",
                "jkl": null
            },
            {
                "abc": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "def": null,
                "ghi": "2000000000",
                "jkl": null
            },
            {
                "abc": null,
                "def": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "ghi": null,
                "jkl": "870000000"
            },
            {
                "abc": "0x0000000000000000000000000000000000000000",
                "def": null,
                "ghi": "870000000",
                "jkl": null
            },
            {
                "abc": "0xf9450d254a66ab06b30cfa9c6e7ae1b7598c7172",
                "def": null,
                "ghi": "100",
                "jkl": null
            }
        ]
    }
}
```

![image](https://user-images.githubusercontent.com/5861379/103096304-e515d380-4646-11eb-800d-d4d55512e6e0.png)

I tested this PR by merging #35 into this branch, and indeed, the output is broken:

![image](https://user-images.githubusercontent.com/5861379/103098955-00d1a780-4650-11eb-9817-ea8a9df76f38.png)